web/src/components/pages/profile/info/info.css

```diff
@@ -124,6 +132,12 @@
                 text-decoration: none;
             }
         }
+
+        a.terms {
```

Comment:
I'm writing some pretty loosey-goosey CSS, so let me know if there's a style guide / ordering preference / etc that you were adhering to. One example that comes to mind is that I'm omitting leading `&`s where possible.

Reply:
I've been a loose goose myself, as you might've noticed. Had some ambitions about that a year ago, but that's about it in terms of style guide:
https://github.com/mozilla/voice-web/issues/912

Reply:
Oh cool, thanks for the link!